asset-transfer-basic/rest-api-typescript/scripts/generateEnv.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+${AS_LOCAL_HOST:=true}
+
+: "${TEST_NETWORK_HOME:=../..}"
+: "${CONNECTION_PROFILE_FILE_ORG1:=${TEST_NETWORK_HOME}/organizations/peerOrganizations/org1.example.com/connection-org1.json}"
+: "${CERTIFICATE_FILE_ORG1:=${TEST_NETWORK_HOME}/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/signcerts/Admin@org1.example.com-cert.pem}"
+: "${PRIVATE_KEY_FILE_ORG1:=${TEST_NETWORK_HOME}/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/keystore/priv_sk}"
+
+: "${CONNECTION_PROFILE_FILE_ORG2:=${TEST_NETWORK_HOME}/organizations/peerOrganizations/org2.example.com/connection-org2.json}"
+: "${CERTIFICATE_FILE_ORG2:=${TEST_NETWORK_HOME}/organizations/peerOrganizations/org2.example.com/users/Admin@org2.example.com/msp/signcerts/Admin@org2.example.com-cert.pem}"
+: "${PRIVATE_KEY_FILE_ORG2:=${TEST_NETWORK_HOME}/organizations/peerOrganizations/org2.example.com/users/Admin@org2.example.com/msp/keystore/priv_sk}"
+
+
+cat << ENV_END > .env
+LOG_LEVEL=info
+
+PORT=3000
+
+RETRY_DELAY=3000
+
+MAX_RETRY_COUNT=5
+
+HLF_CERTIFICATE_ORG1="$(cat ${CERTIFICATE_FILE_ORG1} | sed -e 's/$/\\n/' | tr -d '\r\n')"
+
+HLF_PRIVATE_KEY_ORG1="$(cat ${PRIVATE_KEY_FILE_ORG1} | sed -e 's/$/\\n/' | tr -d '\r\n')"
+
+HLF_CERTIFICATE_ORG2="$(cat ${CERTIFICATE_FILE_ORG2} | sed -e 's/$/\\n/' | tr -d '\r\n')"
+
+HLF_PRIVATE_KEY_ORG2="$(cat ${PRIVATE_KEY_FILE_ORG2} | sed -e 's/$/\\n/' | tr -d '\r\n')"
+
+HLF_COMMIT_TIMEOUT=3000
+
+HLF_ENDORSE_TIMEOUT=30
```

Comment:
Are these in different units?
Good to put the units as a comment to make it clear.

Reply:
I thought they were the same unit but I'll check

generateEnv.sh is a convenience script for running the sample and the units are documented in config.ts - I'd prefer to avoid duplicating the comments but it would definitely help to have a comment at the top of the generated file pointing to config.ts, which I think relates to one of Matthew's comments

Reply:
They are both seconds
hyperledgendary/fabric-rest-sample#54